.github/workflows/apiserver-test.yaml

```diff
@@ -136,13 +136,13 @@ jobs:
       - name: Setup K3d (Hub)
         uses: nolar/setup-k3d-k3s@v1.0.8
         with:
-          version: v1.18
+          version: v1.20
```

Comment:
let's make the version as a global value in action.

The same below

Reply:
Yes, this version is only for test now.